packages/bp/src/sdk/botpress.d.ts

```diff
@@ -990,6 +991,12 @@ declare module 'botpress/sdk' {
     sessionTimeoutInterval: string
   }
 
+  export interface MessagingConfig {
+    id: string
+    token: string
+    channels: { [channelName: string]: any }
```

Comment:
Please add comments with a link to the documentation where we can find the typings for channels. Also, don't forget to update `bot.config.ts` with the same typings, since that's what we use to generate the schema for the code editor